### DISCUSSION TITLE
Prisma+PNPM  workspaces

### DISCUSSION THREAD

**Discussion by Fractal-Tess** [prisma/prisma#15051] [Q&A] [answered]
Hello there,

I have been trying to set-up a monorepo for my hobby project. I landed on `PNPM` (wanted to try it out and i really like it) and `prisma` (using it for the first time but I love how simple and powerful it is).
However, there is an issue when trying to use the generated `PrismaClient` with pnpm workspaces.
I have setup two example repositories that showcase a working yarn setup and a non-working pnpm setup. 
Here is the repo https://github.com/Fractal-Tess/prisma-yarn-pnpm

Also, I would like to note that I have tried to generate the client output to different node_module folders via the output option like this:
``` output = "../node_modules/.prisma/client ```. 
So far I have tried a number of combinations when generating the client to different outputs, including, to `/app/root/node_modules`, to `/packages/prisma/node_modules`, or to `/node_modules`.

Anyway, I really like prisma and pnpm and would love to continue using them.

**Comment by austincrim** [comment, marked as answer]
Hey @Fractal-Tess 👋,

Could you elaborate on the issue you are seeing? Is there any error output you can share?

There is a potential workaround outlined in [this issue comment](https://github.com/prisma/prisma/issues/5340#issuecomment-896826851). Could you try following those steps to see if it resolves your issue?

**Reply by vladi-stevanovic** [reply]
👋 Hello @Fractal-Tess I'm following up to see if the workaround suggestion by Austin worked? Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved!

**Reply by Fractal-Tess** [reply]
Hey there,

I'm not sure if the issue was fixed or if it was an issue in the first place. Perhaps I was misusing it to begin with. I've opted to switch package managers from pnpm to yarn and forget about all this pain.

**Reply by vladi-stevanovic** [reply]
Thank you for letting us know! If you ever go back to pnpm in the future and you still have issues, please let us know!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
